class.c

```diff
@@ -1375,7 +1375,7 @@ move_refined_method(ID key, VALUE value, void *data)
             RB_OBJ_WRITE(me, &me->def->body.refined.orig_me, NULL);
             new_me = rb_method_entry_clone(me);
             rb_method_table_insert(klass, tbl, key, new_me);
-            rb_method_entry_copy(me, orig_me);
+            rb_method_entry_copy(me, orig_me, true);
```

Comment:
We shouldn't need to pass `complemented` to `rb_method_entry_copy`. We can determine that using `METHOD_ENTRY_COMPLEMENTED(src}` in `rb_method_entry_copy`.